cloudconfig/containerinit/containerinit_fromhost.go

```diff
@@ -0,0 +1,162 @@
+package containerinit
```

Comment:
This would better be named as cloudinit_fromhost.go
The single exported method in this file is to read the cloud init data

Reply:
moved cloudconfig/containerinit/containerinit_fromhost.go to cloudconfig/machinecloudconfig.go